### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by demoonism**
Change wording to avoid confusion

## What changes were proposed in this pull request?
Change the wording in the installation doc to avoid confusion
## How was this patch tested?
Peer reviewed

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ source, visit [Building Spark](building-spark.html).
 
 Spark runs on both Windows and UNIX-like systems (e.g. Linux, Mac OS). It's easy to run
 locally on one machine --- all you need is to have `java` installed on your system `PATH`,
-or the `JAVA_HOME` environment variable pointing to a Java installation.
+or the `JAVA_HOME` environment variable pointing to an existing Java installation.
```
Comment (srowen) [inline docs/index.md]: What's the ambiguity here?

Comment (demoonism) [reply docs/index.md]: A very subtle change here. Trying to make this more "noobie friendly" by implying that user needs to set up JRE before running spark.

Comment (HyukjinKwon) [reply docs/index.md]: I din't think this line is confusing ..

**Comment by demoonism** [comment]
Closing as the change is subtle.

**Pull request closed without merging**